src/main/java/com/derongan/minecraft/guiy/gui/FillableElement.java

```diff
@@ -72,6 +72,23 @@ public void removeElement(int x, int y) {
         elements[x][y] = null;
     }
 
+    /**
+     * Removes all references to a specific element from the contents of the elements grid.
+     *
+     * @param element The element to be removed.
+     */
+    public void removeElement(Element element) {
+        for (int x = 0; x < width; x++) {
+            for (int y = 0; y < height; y++) {
+                if (elements[x][y].equals(element)) {
```

Comment:
Elements don't define equals so this will almost always fall back on an instance check. Is this intended?

Reply:
Hmm, no the idea would be to remove a specific element without having to get its position first. Perhaps it could also be useful to remove all elements of a certain type, but that wasn't the intent here.